### ISSUE TITLE
The Official Peep Model

### ISSUE THREAD

**Issue by oli414** [OpenRCT2/OpenGraphics#17]
If I were to make a guess, more than a quarter of all the sprites are peep sprites. The peep sprites have hundreds of animation frames for moving around the park, as well as separately rendered peep sprites for each individual ride. On top of that, the staff sprites use the peep as a base and expand upon that to create visually distinct characters with their own unique animations.

Almost all ride replacement models will have the need for a peep model. To ensure consistency we will have to provide a peep model that can be used for all use cases.

I've made an approximation of the peep model that Saad has turned into a more [professionally rigged character](https://github.com/OpenRCT2/OpenGraphics/pull/11) to support animations in #11. This peep model was created from a mock-up but we need to discuss its shape and form. And eventually release an official OpenRCT2 peep model that can be used for the OpenGraphics project, as well as custom rides.

**Comment by oli414** [comment]
I think that for the peep model it would be a good idea to stick close to the original as it is one of the defining factors. The model Saad has optimized and animated is really close to the shape of the original, I think the model by Saad would be a good base but could use some input from someone a bit more experienced with 3D modeling characters for a more natural shape.

For the time being, we'd maintain the currently existing variations but in the future, we might want to start discussing the possibility of offering more representation in hair colors and skin tones. If we can ensure that the hair and skin's colors from the palette are unique to those regions then this could easily be done at a later stage without the need for re-rendering sprites.

**Comment by duncanspumpkin** [comment]
I don't see having different skin/hair working with how remapping works on coaster cars. You would have to introduce another couple remapable colours and by then we are talking reworking the whole rendering engine.

**Comment by zrowny** [comment]
I could take a better look at the rendering system and how remapping works, but it wouldn't be impossible of a change to have a special section just for peep sprites that remaps the hair color/skin color separately. That would work on the original graphics too, so really this should be a discussion/feature request for the OpenRCT2 repo

**Comment by oli414** [comment]
Yeah that was my idea as well, the hair and skin tone stay each stay within one color group, so it might be possible to remap those. But that's probably something to discuss later.

What does everyone think of this shape for the peep?

![image](https://user-images.githubusercontent.com/2348094/117586505-ea1c8080-b118-11eb-9b4d-ed920cbdc152.png)

![image](https://user-images.githubusercontent.com/2348094/117586538-0e785d00-b119-11eb-9202-24cd55481f5b.png)

It'll probably need some work, but I'll mostly like to hear about the opinions about the general shape, and style. Like the large eyes, and nose.

Amazing Earl back in the day made the following peep for his custom rides:
![image](https://user-images.githubusercontent.com/2348094/117586599-7169f400-b119-11eb-82a3-c13d0b81731d.png)
A pioneer in the stuff we're attempting to do with this project. The character shares a similar shape to the one by me and Saad so for the rough shape I think we're on the right track.

I decided to add the nose after diving deep into the peep sprites and finding that characters had a pixel sticking out of their heads when looking diagonally, making me believe that the peep model might have had the large nose that appears in the 2D icons of peep faces. But we can't really know for sure.

It might help to understand the peep throughout development:
http://www.nicscorner.com/evo_of_rct.htm
In the first image displaying peeps they had cones for noses:
![image](https://user-images.githubusercontent.com/2348094/117586775-85622580-b11a-11eb-9cb5-bd1d31d90cce.png)
Later the peeps were made a bit more human, and given blond hair:
![image](https://user-images.githubusercontent.com/2348094/117586788-9b6fe600-b11a-11eb-8925-92f2c8580939.png)
Eventually, it looks like the peeps were given some more volume along with some changes to the face that made them a bit more cartoony:
![image](https://user-images.githubusercontent.com/2348094/117586808-d07c3880-b11a-11eb-9363-1131fe1f842a.png)

Here're some sprites that didn't appear in the game but that clearly show a nose when heading diagonally
![image](https://user-images.githubusercontent.com/2348094/117586833-eb4ead00-b11a-11eb-8161-6829f9d536a6.png)
![image](https://user-images.githubusercontent.com/2348094/117586842-fefa1380-b11a-11eb-8d43-021a15ebf0c8.png)

What do you folks think?

**Comment by keiyakins** [comment]
I do think you're right about the nose. Facial features need to be exaggerated to be visible at all, so a big ol' nose and giant black eyes make sense, unless someone really wants to go through and manually place pixel eyes on every last sprite.

It'd be nice if we could make our peep model a bit more androgynous than the original - we're probably only going to have the one so it needs to be as Generic Human as possible, and the classic one reads quite solidly masculine.

**Comment by oli414** [comment]
> I do think you're right about the nose. Facial features need to be exaggerated to be visible at all, so a big ol' nose and giant black eyes make sense, unless someone really wants to go through and manually place pixel eyes on every last sprite.

Oh I wasn't trying to say that the nose was added manually, I just suspect that the original peep model had a large nose. So I'm just trying to get the opinions about that

> It'd be nice if we could make our peep model a bit more androgynous than the original - we're probably only going to have the one so it needs to be as Generic Human as possible, and the classic one reads quite solidly masculine.

Alright, what do you think would help to make it more generic?

**Comment by mrpenguinb** [comment]
> > It'd be nice if we could make our peep model a bit more androgynous than the original - we're probably only going to have the one so it needs to be as Generic Human as possible, and the classic one reads quite solidly masculine.
> 
> Alright, what do you think would help to make it more generic?

The shoulders are too broad, and maybe shrink the waist a bit. That would help make it more generic IMO
